src/python/pants/backend/python/dependency_inference/import_parser_test.py

```diff
@@ -84,6 +84,12 @@ def test_normal_imports(rule_runner: RuleRunner) -> None:
         import demo
         from project.demo import Demo
         from project.demo import OriginalName as Renamed
+        import pragma_ignored  # pants: ignore
+        from also_pragma_ignored import doesnt_matter  # pants: ignore
+        from multiline_import import (
+            not_ignored,
+            should_be_ignored_but_isnt  # pants: ignore
```

Comment:
This is a bummer. The node starts at the `from ..` line and ends at the `}` line and there's no in-between info.

We could suss out which line each import comes from, but there'd be thorns. Consider:
```python
from multiline_import import (
   import A as B
   import B as C  # pants: ignore
   import C as D
)
```

simply searching for "B" or "C" would result in incorrect lines...

Reply:
I have an idea for how to fix this. sit tight

Reply:
Nevermind :) We should just document the behavior and pray isort doesn't make this too annoying.

Reply:
JK got it working :tada: